src/main/java/com/raisetech/inventoryapi/controller/InventoryProductController.java

```diff
@@ -76,4 +78,9 @@ public InventoryProduct findInventoryById(@PathVariable(value = "id") int id) th
         inventoryProductService.deleteInventoryById(id);
         return ResponseEntity.ok(Map.of("message", "Successful operation"));
     }
+
+    @GetMapping("/inventory-products/current-inventories")
+    public List<CurrentInventory> getCurrentInventories() {
```

Comment:
質問です。
Currentをつける意味ってありますか？
在庫なのであればInventoryでよいと思いました。
```suggestion
    public List<Inventory> getCurrentInventories() {
```
Entityが増えてきているのでそれぞれの関係や役割をクラス図などで整理するとよいかもしれません。
<img width="269" alt="スクリーンショット 2024-08-06 22 46 00" src="https://github.com/user-attachments/assets/a55effcd-8719-4423-aa9e-52f652197968">

Reply:
コメントありがとうございます！
他のクラスがInventory○○と既にInventoryを使っていたため敢えてCurrentで「現在庫」とという意味でつけてみました。が、ご指摘のように在庫という意味では「Inventory」で十分と言われれば、それもそうと考えますので、Entityの命名を変更します。
1b77deca6e816221186dc28623ea7ac8d28e8839

> クラス図などで整理するとよい

アドバイスいただき有難うございます。
こちら検討します。